src/main/kotlin/no/nav/familie/ks/sak/kjerne/personopplysninggrunnlag/PersonopplysningGrunnlagService.kt

```diff
@@ -46,14 +49,32 @@ class PersonopplysningGrunnlagService(
         )
     }
 
+    fun oppdaterPersonopplysningGrunnlag(behandling: Behandling, søknadDto: SøknadDto) {
+        val eksisterendePersonopplysningGrunnlag =
+            personopplysningGrunnlagRepository.findByBehandlingAndAktiv(behandling.id)
+                ?: throw Feil("Det finnes ikke noe aktivt personopplysningsgrunnlag for ${behandling.id}")
```

Comment:
```suggestion
    fun hentAktivPersonopplysningGrunnlag(behandlingId: Long): PersonopplysningGrunnlag =
 personopplysningGrunnlagRepository.findByBehandlingAndAktiv(behandling.id)
                 ?: throw Feil("Det finnes ikke noe aktivt personopplysningsgrunnlag for ${behandling.id}")
```

Denne kan sikkert trekkes ut til en metode, ser i ba-sak at det ikke er siste gang vi kommer til å ta samme sjekk 😛